### PR TITLE
EZP-30648: Values of checkboxes are not displayed properly in Content  Type view

### PR DESCRIPTION
**Pull request by pawbuj**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30648](https://jira.ez.no/browse/EZP-30648)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

fix

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig [hunk 1]
```diff
@@ -64,7 +64,7 @@
         </tr>
         <tr class="ez-table__row">
             <td class="ez-table__cell">{{ "content_type.container"|trans|desc("Container") }}</td>
-            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer)|desc("{0}No|{1}Yes") }}</td>
+            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer|number_format)|desc("{0}No|{1}Yes") }}</td>
```
Comment (adamwojs) [inline src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig]: IMHO `? 1 : 0` seems to be a cleaner approach:

```suggestion
            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer ? 1 : 0)|desc("{0}No|{1}Yes") }}</td>
```

File: src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig [hunk 2]
```diff
@@ -85,7 +85,7 @@
                     {{ "content_type.default_availability.help"|trans|desc("Default availability in primary language, if translation is absent") }}
                 </p>
             </td>
-            <td class="ez-table__cell">{{ 'content_type.default_availability.value'|transchoice(content_type.defaultAlwaysAvailable)|desc("{0} Not available|{1} Available") }}</td>
+            <td class="ez-table__cell">{{ 'content_type.default_availability.value'|transchoice(content_type.defaultAlwaysAvailable|number_format)|desc("{0} Not available|{1} Available") }}</td>
```
Comment (adamwojs) [inline src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig]:
```suggestion
            <td class="ez-table__cell">{{ 'content_type.default_availability.value'|transchoice(content_type.defaultAlwaysAvailable ? 1 : 0)|desc("{0} Not available|{1} Available") }}</td>
```

File: src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig [hunk 3]
```diff
@@ -64,7 +64,7 @@
         </tr>
         <tr class="ez-table__row">
             <td class="ez-table__cell">{{ "content_type.container"|trans|desc("Container") }}</td>
-            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer)|desc("{0}No|{1}Yes") }}</td>
+            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer ? 1 : 0)|desc("{0}No|{1}Yes") }}</td>
```
Comment (alongosz) [inline src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig]: This filter is deprecated, can we use `trans` here [as suggested in the doc](https://symfony.com/doc/current/reference/twig_reference.html#transchoice)?

Comment (alongosz) [reply src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig]: Ok, I'm ready to trade approve for a follow-up JIRA issue number :P

Comment (pawbuj) [reply src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig]: https://jira.ez.no/browse/EZP-30676

**Pull request merged**